### PR TITLE
Laravel 8 and Backpack 5.x Upgrade

### PR DESCRIPTION
**Pull request by fsmonter**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@ yarn-error.log
 core
 
 .idea
+
+auth.json
```
Comment (fsmonter) [inline .gitignore:37]: Every environment has to add an `auth.json` entry to authenticate the backpack/pro package.

**Comment by fsmonter** [comment]
Hey @adammparker, upgrade tested on staging and is ready for production. Let me know what you think!

**Review by adammparker — CHANGES_REQUESTED** [review]
Only one code-level change I think, I poked around a little bit on staging and things look pretty much the same as before. Did you test some of the admin functionality like actually performing create/edit etc. functions?

File: config/backpack/base.php [hunk 2]
```diff
@@ -269,4 +291,26 @@
...
+
+    // Backpack will prevent an IP from trying to reset the password too many times,
+    // so that a malicious actor cannot try too many emails, too see if they have
+    // accounts or to increase the AWS/SendGrid/etc bill.
+    //
+    // How many times in any given time period should the user be allowed to
+    // attempt a password reset? Take into account that user might wrongly
+    // type an email at first, so at least allow one more try.
+    // Defaults to 3,10 - 3 times in 10 minutes.
+    'password_recovery_throttle_access' => '3,10',
```
[the remark this answers is not in the document]
Comment (fsmonter) [reply config/backpack/base.php]: Woops yes missed that. It's updated.

**Review by adammparker — APPROVED** [review]

**Pull request merged**